scripts/pre_commit_linter.py

```diff
@@ -1573,14 +1670,15 @@ def main():
     html_tag_and_attribute_messages = _check_html_tags_and_attributes(all_files)
     html_linter_messages = _lint_html_files(all_files)
     linter_messages = _pre_commit_linter(all_files)
+    unused_dependies_messages = _check_for_unused_dependencies(all_files)
```

Comment:
Spell error in dependencies: here and below.

Reply:
Done..!